src/parts/Sidebar/SidebarContent/Navigation/index.tsx

```diff
@@ -149,24 +188,18 @@ const Navigation = ({
                   { 'dark:hover:bg-white': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA },
                   { 'dark:hover:bg-opacity-10': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA }
                 ),
-              'group',
-              'flex',
-              'items-center',
-              'px-2',
-              'py-2',
               onSmallScreen ? 'text-base' : 'text-sm',
-              'font-medium',
-              'rounded-md'
+              textClasses,
+              'font-medium'
+
             )}>
             <navigationItem.icon
               className={clsx(
                 match?.isExact ?
                   textClassesForSelected :
                   textClassesForUnselected,
                 onSmallScreen ? 'mr-4' : 'mr-3',
```

Comment:
@tomjeatt 
I guess this could be included in `navigationIconClasses` instead of being duplicated.

Reply:
@crypto-engineer on reflection, I think I'd prefer to leave it as it is. Both solutions are imperfect, but I think the simpler imperfect solution (a little bit of repeated code) is better than the more complex repeated solution (generating the classes with a function, rather than returning the classes using `cslx` as we're doing now). We won't be retaining this code for very long, as it will be redundant once we've rolled out the currently disabled features.